curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-067.md

```diff
@@ -1,20 +1,41 @@
 ---
-id: 5f3ef6e0b431cc215bb16f55
+id: 5f3ef6e01f288a026d709587
 title: Part 67
 challengeType: 0
 dashedName: part-67
 ---
 
 # --description--
 
-Add a second `p` element below the one with the link and give it the text `123 freeCodeCamp Drive`.
+You can use an `hr` element to display a divider between sections of different content.
+
+First, add an `hr` element between the first `header` element and the `main` element.
```

Comment:
if hr elements are not previously taught, we should mention that they are self-closing

Reply:
It's mentioned in the first test, but I included a note in the instructions too. 🙂